test/runtests.jl

```diff
@@ -1053,8 +1053,6 @@ end
 
     @test copy(m) ≡ m
     @test copy(D) ≡ D
-    @test LinearAlgebra.copy_oftype(m, Int) ≡ Eye{Int}(10)
-    @test LinearAlgebra.copy_oftype(D, Float64) ≡ Diagonal(Fill(2.0,10))
```

Comment:
I'm not sure what this tests. It was added as part of https://github.com/JuliaArrays/FillArrays.jl/pull/142. The `copy` for `Eye` and `Diagonal` still works, so I think it's safe to remove these two